src/Lime.Transport.SignalR/ClientSignalRTransport.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Sockets;
+using System.Net.WebSockets;
+using System.Threading;
+using System.Threading.Channels;
+using System.Threading.Tasks;
+using Lime.Protocol;
+using Lime.Protocol.Network;
+using Lime.Protocol.Serialization;
+using Microsoft.AspNetCore.SignalR.Client;
+
+namespace Lime.Transport.SignalR
+{
+    /// <summary>
+    /// Provides methods for client nodes to communicate with a server using SignalR as the underlying transport mechanism.
+    /// This class cannot be inherited.
```

Comment:
Comment `this class cannot be inherited` is redundant - the `sealed` modifier already tell us this. If this is important, explain **why** you have marked it as sealed instead.

Reply:
@andreminelli Oh sure, I also agree that it is quite trivial to know that the class is sealed, even when reading documentation. I was only following framework conventions.